test/behavior.zig

```diff
@@ -143,6 +143,7 @@ test {
     _ = @import("behavior/bugs/13785.zig");
     _ = @import("behavior/bugs/14854.zig");
     _ = @import("behavior/bugs/15778.zig");
+    _ = @import("behavior/bugs/16007.zig");
```

Comment:
Please do not add new `bugs/XXX.zig` files, instead put the tests in one of the existing categories.

Reply:
I'm sorry. Tests moved to `if.zig`